src/Symfony/Component/Translation/Command/XliffLintCommand.php

```diff
@@ -126,14 +126,24 @@ private function validate($content, $file = null)
             // otherwise, both '____.locale.xlf' and 'locale.____.xlf' are allowed
             // also, the regexp matching must be case-insensitive, as defined for 'target-language' values
             // http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html#target-language
-            $expectedFilenamePattern = $this->requireStrictFileNames ? sprintf('/^.*\.(?i:%s)\.xlf/', $normalizedLocale) : sprintf('/^(.*\.(?i:%s)\.xlf|(?i:%s)\..*\.xlf)/', $normalizedLocale, $normalizedLocale);
+            $expectedFileExtensionPattern = $this->requireStrictFileNames ? '/^.*\.xlf/' : '/^.*\.(xlf|xliff)/';
 
-            if (0 === preg_match($expectedFilenamePattern, basename($file))) {
+            if (0 === preg_match($expectedFileExtensionPattern, basename($file))) {
                 $errors[] = [
                     'line' => -1,
                     'column' => -1,
-                    'message' => sprintf('There is a mismatch between the language included in the file name ("%s") and the "%s" value used in the "target-language" attribute of the file.', basename($file), $targetLanguage),
+                    'message' => sprintf('Invalid file extension used in strict mode ("%s").', basename($file)),
                 ];
+            } else {
```

Comment:
why `else`? shouldn't you get both errors when both the extension and the content is wrong?